docs/src/index.js

```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import { HashRouter, Route, Switch } from 'react-router-dom';
 import { render } from 'react-dom';
-import 'gestalt/dist/gestalt.css';
+import 'gestalt/dist/gestalt-future.css';
```

Comment:
I'd like to start dogfooding the new css in the docs. This means colors won't look right for IE11 in the docs. I *could* fix this by generating the big css file that includes both vars and fallbacks or can we say IE11 isn't supported in the docs?

Reply:
@jennyscript would there be a way to make this import conditional? We recently added support to test the docs in IE11 #957 which is useful to test IE specific docs.

An alternative is to use something like: https://www.npmjs.com/package/css-vars-ponyfill which we could also include conditionally (only for IE11)

Reply:
I added a polyfill for the docs (ended up using a different one for easier set up). The docs only _kind of_ work in IE11 right now, but as far as I could tell colors are showing up correctly in the docs.